docs/features/highlight.md

```diff
@@ -0,0 +1,23 @@
+# Highlight markdown's code blocks
+
+## Overview
+
+The component internally uses [`highlight.js`](https://highlightjs.org/) to highlight the markdown code blocks. To reduce the size of the component, it only uses the configuration for 3 languages: `json`, `yaml` and `bash`. The rest are rendered as plain text. However, the package provides the instance of the [`highlight.js`](https://highlightjs.org/) outside the package with which you can, for example, add a custom language configuration.
```

Comment:
🤔 Why is it you include those 3 specific languages?

Reply:
If you have description in md, you probably will use (at least 90% 😄 ) `json`, `yaml` or `bash` language to describe config for server in json/yaml, maybe some bash script to make curl etc - they are the most used languages ​​when it comes to describing something so they are added by default. Here I am based on the probability of use. Additionally, `json` is used for rendering examples for messages and these three languages and they don't add that much kbs to the package.

Reply:
Okay fair enough 🙂